src/gui/torrentcontentmodel.cpp

```diff
@@ -39,299 +39,297 @@
 {
-  Q_ASSERT(m_filesIndex.size() == (int)fprio.size());
-  // XXX: Why is this necessary?
-  if (m_filesIndex.size() != (int)fprio.size())
-    return;
-
-  emit layoutAboutToBeChanged();
-  for (int i = 0; i < fprio.size(); ++i) {
-    m_filesIndex[i]->setPriority(fprio[i]);
-  }
-  emit dataChanged(index(0, 0), index(rowCount(), columnCount()));
+    Q_ASSERT(m_filesIndex.size() == (int)fprio.size());
+    // XXX: Why is this necessary?
+    if (m_filesIndex.size() != (int)fprio.size())
+        return;
+
+    emit layoutAboutToBeChanged();
+    for (int i = 0; i < fprio.size(); ++i)
+        m_filesIndex[i]->setPriority(fprio[i]);
+    emit dataChanged(index(0, 0), index(rowCount(), columnCount()));
 }
 
 QVector<int> TorrentContentModel::getFilePriorities() const
 {
-  QVector<int> prio;
-  prio.reserve(m_filesIndex.size());
-  foreach (const TorrentContentModelFile* file, m_filesIndex) {
-    prio.push_back(file->priority());
-  }
-  return prio;
+    QVector<int> prio;
+    prio.reserve(m_filesIndex.size());
+    foreach (const TorrentContentModelFile* file, m_filesIndex)
+        prio.push_back(file->priority());
+    return prio;
 }
 
 bool TorrentContentModel::allFiltered() const
 {
-  foreach (const TorrentContentModelFile* fileItem, m_filesIndex) {
-    if (fileItem->priority() != prio::IGNORED)
-      return false;
-  }
-  return true;
+    foreach (const TorrentContentModelFile* fileItem, m_filesIndex)
+        if (fileItem->priority() != prio::IGNORED)
+            return false;
+    return true;
 }
 
 int TorrentContentModel::columnCount(const QModelIndex& parent) const
 {
-  if (parent.isValid())
-    return static_cast<TorrentContentModelItem*>(parent.internalPointer())->columnCount();
-  else
-    return m_rootItem->columnCount();
+    if (parent.isValid())
+        return static_cast<TorrentContentModelItem*>(parent.internalPointer())->columnCount();
+    else
+        return m_rootItem->columnCount();
 }
 
 bool TorrentContentModel::setData(const QModelIndex& index, const QVariant& value, int role)
 {
-  if (!index.isValid())
-    return false;
-
-  if (index.column() == 0 && role == Qt::CheckStateRole) {
-    TorrentContentModelItem *item = static_cast<TorrentContentModelItem*>(index.internalPointer());
-    qDebug("setData(%s, %d", qPrintable(item->name()), value.toInt());
-    if (item->priority() != value.toInt()) {
-      if (value.toInt() == Qt::PartiallyChecked)
-        item->setPriority(prio::MIXED);
-      else if (value.toInt() == Qt::Unchecked)
-        item->setPriority(prio::IGNORED);
-      else
-        item->setPriority(prio::NORMAL);
-      // Update folders progress in the tree
-      m_rootItem->recalculateProgress();
-      emit dataChanged(this->index(0,0), this->index(rowCount()-1, columnCount()-1));
-      emit filteredFilesChanged();
+    if (!index.isValid())
+        return false;
+
+    if (index.column() == 0 && role == Qt::CheckStateRole) {
```

Comment:
>Please add parentheses around subexpressions.

@sledgehammer999 
Please add the following config to your [uncrustify script](https://gist.github.com/sledgehammer999/1cb1d9224d7fec8fa632)
``` 
mod_full_paren_if_bool	=  true
```
I tested that this will fix the problem.